templates/msgext/csharp/default/GettingStarted.txt

```diff
@@ -13,27 +13,17 @@ Quick start
 
 Before you run or Start Debugging for the first time, follow these steps:
 
-1. If your project only contains a Tab capability, skip to step #3.
```

Comment:
Why do we move ME template from bot folder to msgext?

Reply:
Because in the future msgext is a indepent capability, compare with bot.
Since message extension is no longer including bot, also bot no longer including msgext. 
I think create a new folder named msgext is make sense.

Reply:
Also, we plan to move msgext (vs code version) indenpent too.

Reply:
As discussed, change the folder path